lib/util/PDFDocumentWithTables.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const PDFDocument = require('pdfkit')
+
+/**
+ * This class extends the pdfkit functionality to provide a tables method, which
+ * will write tableData (2-d array) to the pdf doc.
+ *
+ * See https://www.andronio.me/2017/09/02/pdfkit-tables/ for more info.
+ */
+class PDFDocumentWithTables extends PDFDocument {
+  table (table, arg0, arg1, arg2) {
```

Comment:
The arguments for this method need to be renamed. Or, if the use is ambivalent, provide JSDoc for this method.